src/ui.cpp

```diff
@@ -629,17 +627,18 @@ void UpdateDialog::OnClose(wxCloseEvent&)
 }
 
 
-void UpdateDialog::OnSkipVersion(wxCommandEvent&)
+void UpdateDialog::OnRemindOneHour(wxCommandEvent&)
 {
-    Settings::WriteConfigValue("SkipThisVersion", m_appcast.Version);
+    static const int ONE_HOUR_IN_SECONDS = 3600;
+    Settings::WriteConfigValue("UpdateInterval", ONE_HOUR_IN_SECONDS);
```

Comment:
Is there a default for UpdateInterval? I.e. what happens when neither of the Remind buttons are pushed and the X (close window) is used instead?